IMAGE-CLASSIFICATION/ResNet-PyTorch/resnet.md

```diff
@@ -0,0 +1,159 @@
+# A short summary and implementation on the ResNet model(CIFAR-10 Dataset)
+
```

Comment:
rename resnet.md file to README.md and instruction to run the model  something like `python3 main.py --block_type "basic" ` and give reference to the paper